CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# HERMES development workflow
+
+## Preamble
+
+> **Branching is cheap!**
+>
+> Work in small packets, put up small pull requests!
+>
+> Aim for quick turnaround times!
+
+## Branching
+
+We loosely follow a mixture of [GitHubFlow](https://docs.github.com/en/get-started/quickstart/github-flow) and [GitFlow](https://nvie.com/posts/a-successful-git-branching-model/):
+
+`main`
+    - Stable branch
+    - Merges come only from `develop` or a hotfix branch (i.e., when something needs to be fixed in "production")
+
+`develop`
+    - Unstable development branch
+    - Last "stable" is tagged: tag before attempting to break something
+
+`feature/<describe-feature>` (including issue id when exists: `feature/62-fix-broken-thing`)
```

Comment:
```suggestion
`<issue id>-<describe-feature>`

1. Open issue first
2. Use max 2-3 words for description
3. Concatenate with dashes
```

Reply:
**suggestion**:
- I really got comfortable with the issue first thing with Dataverse.
- I had problems in the past with "/" in branch names
- I like it short
- I like switching the branch by typing the number and use tab complete

Reply:
Thanks.

>     * I really got comfortable with the issue first thing with Dataverse.

When there's a good scope for an issue, I don't disagree, hence the inclusion of id in branch names.
In my experience though, what this leads to is either scope creep in issues, or modeling issue scope based on agile changes in PRs.

If there is defined scope for an issue, there's a good chance that it should be an ADR, not an issue, especially on the road to MVP.

>     * I had problems in the past with "/" in branch names

What problems have you had?
 
>     * I like it short

Short is good, and also relative. The prefixes allow for better overview when there are many short-lived branches (the way it should be).

>     * I like switching the branch by typing the number and use tab complete

Tab-complete is still possible with these prefixes :shrug:.

Reply:
I had problems a few years back with some fancy Git client and referencing branch names with a "/" in it. Maybe just a habit from back then that sticked.

I'll just resolve this and bow my head - it really is somewhat bikeshedding, so let's have the green one.

Reply:
Nice one. Also, we can always iterate if we feel something doesn't work. I'll re-request a review for you to approve ;).